samples/outreach/blogs/segmentation_blogpost/5_Image_Segmentation.ipynb

```diff
@@ -0,0 +1,1840 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "cl79rk4KKol8"
+   },
+   "source": [
+    "# Finding visual objects in images - Image Segmentation with tf.keras \n",
+    "In this tutorial we will learn how to segment images. **Segmentation** is the process of generating pixel-wise segmentations giving the class of the object visible at each pixel. For example, we could be identifying the location and boundaries of people within an image or identifying cell nuclei from an image. Formally, image segmentation refers to the process of partitioning an image into a set of pixels that we desire to identify (our target) and the background. \n",
```

Comment:
Could you add the standard set of github/colab buttons up top?

Like here:

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/eager/python/examples/generative_examples/image_captioning_with_attention.ipynb